x-pack/metricbeat/module/airflow/_meta/fields.yml

```
@@ -0,0 +1,21 @@
+- key: airflow
+  title: "Airflow"
+  description: >
+    Airflow module
+  release: ga
```

Let's go first as beta :slightly_smiling_face: 
```suggestion
  release: beta
```